### ISSUE TITLE
Typo in shortcuts overview page

### ISSUE THREAD

**Issue by pat-s** [rstudio/rstudio#6281]
https://support.rstudio.com/hc/en-us/articles/200711853-Keyboard-Shortcuts

for "Goto File/Function" -> "Go to File/Function"

**Comment by gtritchie** [comment]
There are quite a few out-of-date or missing shortcuts on this page. I will use this issue to track fixing the entire page to match the recent work I did on the built-in shortcuts page: #6242

**Comment by gtritchie** [comment]
I've fixed this and several other issues.
